File: Automation/currency.py
import requests

def convert_currency(amount, from_currency, to_currency):
    # API endpoint for retrieving exchange rates
    url = f"https://api.exchangerate-api.com/v4/latest/{from_currency}"

    # Send a GET request to the API
    response = requests.get(url)
    data = response.json()

    if response.status_code == 200:
        exchange_rate = data["rates"][to_currency]

File: Automation/currency.go
package main

import (
  "fmt"
  "net/http"
)

func currencyConverter(amount float64, fromCurrency string, toCurrency string) float64 {
	// Get the exchange rate from the Open Exchange Rates API.
	url := "https://api.exchangeratesapi.io/latest?base=" + fromCurrency
	resp, err := http.Get(url)
	if err != nil {
	  panic(err)
	}

	defer resp.Body.Close()

  data, err := resp.ReadAll()
  if err != nil {
    panic(err)
  }

